### PR TITLE
serial_ftdi: use Sleep() on Win32

### PR DESCRIPTION
**Pull request by neolit123**
### Describe the pull request:

- [ ] Bug fix
- [x] Functional change
- [ ] New feature
- [ ] Code cleanup
- [ ] Build system change
- [ ] Documentation change
- [ ] Language translation

### Pull request long description:

Windows doesn't have nanosleep() unless libwinpthread is used.

Since the nanosleep() usage in serial_ftdi_sleep():
- does not break in case of EINTR
- has input in milliseconds

the WINAPI Sleep() should be a good alternative.

### Changes made:

1) call Sleep() on Win32.

### Related issues:

NONE

### Additional information:

i must admit, i didn't try building with SERIAL_FTDI, so let's see if CI is happy.

### Release note:

NONE

### Documentation change:

NONE

### Mentions:

@dirkhh

### CODE REVIEW

File: core/serial_ftdi.c [hunk 1]
```diff
@@ -107,6 +112,9 @@ static dc_status_t serial_ftdi_sleep (void *io, unsigned int timeout)
 
 	INFO (device->context, "Sleep: value=%u", timeout);
 
+#ifdef __MINGW32__
```
Comment (glance-) [inline core/serial_ftdi.c]: Why `__MINGW32__` and not `WIN32`?

Comment (dirkhh) [reply core/serial_ftdi.c]: A quick grep through our sources seems to indicate that we do indeed use WIN32 in situations like this. Would you please change that, @neolit123

**Review by dirkhh — CHANGES_REQUESTED** [review]
WIN32 change

**Comment by neolit123** [comment]
ACK for consistency. will try to updated later today.

**Comment by neolit123** [comment]
updated to use `_WIN32`.

**Comment by dirkhh** [comment]
UH, does this contain a libdivecomputer update by mistake?

**Comment by neolit123** [comment]
>UH, does this contain a libdivecomputer update by mistake?

sorry, fixed.

**Comment by dirkhh** [comment]
Thanks @neolit123

**Review by dirkhh — APPROVED** [review]

**Pull request merged**